_pages/index.md

```diff
@@ -42,5 +42,5 @@ Le bureau de l'association est actuellement composé de :
 
 # Documents
 
-- [Statuts](/statuts.pdf)
-- [Réglement intérieur](/reglement.pdf)
+- [Statuts](/statuts)
```

Comment:
I wonder if we should keep pdf versions too (auto-generation might be tedious, not sure with jekyll).

Reply:
Quel est l'intérêt d'une version PDF ? Si quelqu'un en a besoin il peut la générer au besoin depuis la version HTML, non ?

Reply:
Genre pour garder l'original de ceux envoyés à la pref ou des choses comme ça. Mais ce n'est peut-être pas le bon endroit pour stocker ça.